doc/source/serve/autoscaling.md

```diff
@@ -0,0 +1,43 @@
+(serve-autoscaling)=
+
+# Serve Autoscaling
+
+This section should help you:
+
+- Understand how Ray Serve autoscaling works
+- Learn how to configure the parameters for your workload
+
+
+## Autoscaling architecture
+Ray Serve autoscaling feature automatically increases or decreases the number of replicas based on the load. 
+
+![pic](https://raw.githubusercontent.com/ray-project/images/master/docs/serve/autoscaling.svg)
+
+- Deployment handle(Client) and worker replica periodically push the stats to the autoscaler.
+- Autoscaler requires serve handle queue metrics and replicas requests metrics to make decision whether to scale (up or down) the replicas.
+- Deployment Handle(Client) keeps polling the replica stats from controller to get the updated replicas information. Serve Handle(Client) will send requests directly to the replica based on the replicas information (Round Robin).
```

Comment:
Let's keep the capitalization and spacing of `ServeHandle` consistent. [The documentation that introduces `ServeHandles`](https://docs.ray.io/en/releases-1.13.0/serve/http-servehandle.html#servehandle-calling-deployments-from-python) uses PascalCase. I suggest we stick with that, but I'm okay with using another format as long as we're consistent.